src/tests/test_method.py

```diff
@@ -304,6 +304,10 @@ def testValueParamsArgs(self):
         self.assertTrue(result[1] == 2)
         self.assertTrue(result[2] == 3)        
 
+    def testNonParamsArrayInLastPlace(self):
```

Comment:
Do you mean the case where there is a params keyword. If so, I believe that's covered by existing tests, eg. testObjectParamsArgs